constellations/orion/src/domains/user.go

```diff
@@ -10,18 +10,20 @@ import (
 var TABLE_USERS = "users"
 
 type User struct {
-	Id         uint         `json:"id"`
-	CreatedAt  time.Time    `json:"-" db:"created_at"`
-	UpdatedAt  time.Time    `json:"-" db:"updated_at"`
-	DeletedAt  sql.NullTime `json:"-" db:"deleted_at"`
-	FirstName  string       `json:"firstName" db:"first_name"`
-	LastName   string       `json:"lastName" db:"last_name"`
-	MiddleName NullString   `json:"middleName" db:"middle_name"`
-	Email      string       `json:"email"`
-	Phone      string       `json:"phone"`
-	IsGuardian bool         `json:"isGuardian" db:"is_guardian"`
-	AccountId  uint         `json:"accountId" db:"account_id"`
-	Notes      NullString   `json:"notes"`
+	Id             uint         `json:"id"`
+	CreatedAt      time.Time    `json:"-" db:"created_at"`
+	UpdatedAt      time.Time    `json:"-" db:"updated_at"`
+	DeletedAt      sql.NullTime `json:"-" db:"deleted_at"`
+	FirstName      string       `json:"firstName" db:"first_name"`
+	LastName       string       `json:"lastName" db:"last_name"`
+	MiddleName     NullString   `json:"middleName" db:"middle_name"`
+	Email          string       `json:"email"`
+	Phone          string       `json:"phone"`
+	IsGuardian     bool         `json:"isGuardian" db:"is_guardian"`
+	AccountId      uint         `json:"accountId" db:"account_id"`
+	Notes          NullString   `json:"notes"`
+	School         NullString   `json:"school" db:"school"`
+	GraduationYear NullUint     `json:"graduationYear" db:"graduation_year"`
```

Comment:
Let's have some validation for graduation years. The graduation year (if not null) should be greater than 2000.
And School (if not null) should have only letters in them.